java/statestore/src/test/java/sleeper/statestore/transactionlog/TransactionLogSnapshotSerDeIT.java

```diff
@@ -52,10 +52,10 @@ void shouldSaveAndLoadPartitionsState() throws StateStoreException {
 
         // When
         TransactionLogPartitionsSnapshotSerDe snapshot = new TransactionLogPartitionsSnapshotSerDe(schema, configuration);
-        snapshot.save(tempDir.toString(), state, 1);
+        String filePath = snapshot.save(tempDir.toString(), state, 1);
 
         // Then
-        assertThat(snapshot.load(tempDir.toString(), 1)).isEqualTo(state);
+        assertThat(snapshot.load(filePath)).isEqualTo(state);
```

Comment:
Maybe for these tests it would be better to create a TransactionLogSnapshot object and pass that into save and load? That way we could put the code to generate the path in TransactionLogSnapshot, and we wouldn't have to have a different method signature between the tests and the real code.